src/components/projects-summary/projects-summary.html

```diff
@@ -88,4 +88,25 @@ <h2 class="secondary">Resources</h2>
       </ul>
     </div>
   </div>
+  <div ng-if="$ctrl.recentlyViewedItems.length">
+    <h2 class="primary">Recently Viewed</h2>
```

Comment:
Actually, going to change this to do the check initially, before iterating on anything:

```
   if (!this.allItems) {
      return;
    }
```